강산아/programmers/week23/숫자의표현.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Author    : Kang San Ah
+ * Date      : 2024.11.08(Fri)
+ * Runtime   : 1 sec
+ * Memory    : 256 MB
+ * Algorithm : 구현
+ */
+
+public class 숫자의표현 {
+
+    public int solution(int n) {
+
+        int answer = 0;
+
+        for(int i=1; i<=n; i++) {
+            int sum = 0;
+            for(int j=i; j<=n; j++) {
+                sum += j;
+
+                if(sum==n) {
+                    answer++;
+                    break;
+                } else if(sum>n) {
+                    break;
+                }
```

Comment:
두 조건다 break 는 공통이니까 이런식으로도 가능할 것 같습니다. (이건 제 스타일)

```suggestion
                if(sum>=n) {
                    if(sum == n) answer ++;
                    break;
                }
```